main.go

```diff
@@ -129,6 +129,7 @@ func router() *mux.Router {
 	router.Handle("/event/list", alice.New(main.apiAuthMiddleware).ThenFunc(main.EventListHandler))
 	router.Handle("/event/delete", alice.New(main.apiAuthMiddleware).ThenFunc(main.EventDeleteHandler))
 	router.Handle("/activist/list", alice.New(main.apiAuthMiddleware).ThenFunc(main.ActivistListHandler))
+	router.HandleFunc("/activist/list_range", main.ActivistInfiniteScrollHandler) // temp unauthed for test
```

Comment:
I forgot to change this to an authed endpoint. I'll make the change before deploying...